.changeset/slimy-pillows-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"eslint-plugin-astro": minor
+---
+
+feat add `astro/missing-directive-value` rule
```

Comment:
We should at least write this with the new rule name.

```suggestion
feat add `astro/missing-client-directive-value` rule
```

Reply:
ah sorry, my mistake.